schemas/endpoints/rebalance.yaml

```diff
@@ -0,0 +1,169 @@
+        in: query
+        description: The upper bound of ongoing leadership movements
+        schema:
+          type: integer
+          format: int32
+          minimum: 1
+      - name: json
+        in: query
+        description: Whether to return in JSON format or not
+        schema:
+          type: boolean
+          default: false
+      - name: skip_hard_goal_check
+        in: query
+        description: Whether to allow hard goal be skipped in proposal generation
+        schema:
+          type: boolean
+          default: false
+      - name: excluded_topics
+        in: query
+        description: Whether to allow hard goal be skipped in proposal generation
+        schema:
+          type: string # topics regex
+          default: null
+      - name: use_ready_default_goals
+        in: query
+        description: Whether to only use ready goals to generate proposal
+        schema:
+          type: boolean
+          default: false
+      - name: data_from
+        in: query
+        description: Whether to calculate proposal from available valid partitions or valid windows
+        schema:
+          type: string
+          default: VALID_WINDOWS
+      - name: verbose
+        in: query
+        description: Return detailed state information
+        schema:
+          type: boolean
+          default: false
+      - name: exclude_recently_demoted_brokers
+        in: query
+        description: Whether to allow leader replicas to be moved to recently demoted broker
+        schema:
+          type: boolean
+          default: false
+      - name: exclude_recently_removed_brokers
+        in: query
+        description: Whether to allow leader replicas to be moved to recently removed broker
+        schema:
+          type: boolean
+          default: false
+      - name: replica_movement_strategies
+        in: query
+        description: Replica movement strategies to use
+        schema:
+          type: array
+          items:
+            type: string
+      - name: ignore_proposal_cache
+        in: query
+        description: Whether to ignore the cached proposal or not
+        schema:
+          type: boolean
+          default: false
+      - name: destination_broker_ids
+        in: query
+        description: Whether to allow leader replicas to be moved to recently demoted broker
+        schema:
+          type: array
+          items:
+            type: string
+      - name: kafka_assigner
+        in: query
+        description: Whether to use Kafka assigner mode to general proposal
+        schema:
+          type: boolean
+          default: false
+      - name: rebalance_disk
+        in: query
+        description: Whether to balance load between brokers or between disks within broker
+        schema:
+          type: boolean
+          default: false
+      - name: review_id
+        in: query
+        description: Review id for 2FA
+        schema:
+          type: string
```

Comment:
[review_id](https://github.com/linkedin/cruise-control/blob/b31598a5b57f0f9d8123a35e4b4a0173d42416fc/cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/RebalanceParameters.java#L74) shoud be an int32 here